packages/cozy-harvest-lib/src/models/ConnectionFlow.spec.js

```diff
@@ -116,6 +117,17 @@ describe('ConnectionFlow', () => {
     const isSubmitting = flow => {
       return flow.getState().running === true
     }
+    beforeAll(() => {
+      watchKonnectorJob.mockReturnValue({on: () => ({})})
+    })
+
+    afterEach(() => {
+      jest.clearAllMocks()
+    })
+
+    afterAll(() => {
+      jest.restoreAllMocks()
```

Comment:
what is the point to use a clearAllMocks before restoreAllMocks:

https://jestjs.io/docs/mock-function-api#mockfnmockrestore

I read here that mockRestore does everything that mockReset does
And mockReset does everything that mockClear deos

Reply:
nit: i recommend to remove clearAllMocks